cfgmgr/teammgr.cpp

```diff
@@ -171,18 +173,30 @@ void TeamMgr::cleanTeamProcesses()
     SWSS_LOG_ENTER();
     SWSS_LOG_NOTICE("Cleaning up LAGs during shutdown...");
 
-    std::unordered_map<std::string, pid_t> aliasPidMap;
+    std::unordered_map<std::string, int> aliasPidMap;
+    uint32_t sleepCounter = 0;
 
     for (const auto& alias: m_lagList)
     {
-        std::string res;
         pid_t pid;
+        if (++sleepCounter % 10 == 0) {
+            // Sleep for 100 milliseconds so as to not overwhelm the netlink
```

Comment:
Do we need to change comment .. is it 100ms or 200ms

Reply:
Also instead of giving a one shot 200ms delay .. can we put 10ms for each delete ? The issues we have seen before is when we delete a large number of Portchannels few stale remains.

So now here you are optimizing the call from shell/cmd approach to system_call() approach which will make it execute faster

I felt the older shell/cmd way of checking for pid/kill was introducing some delay which helped in the cleanup of portchannels.